vipps-recurring-api.md

```diff
@@ -534,12 +537,130 @@ This `sub` is a link between the merchant and the user and can used to retrieve
 the user's details from Vipps Login:
 [`GET:/userinfo/{sub}`](https://vippsas.github.io/vipps-login-api/#/Vipps%20Log%20In%20API/userinfo)
 
-**Please note:** accessing the Login `userinfo` endpoint requires the
-Vipps Login access token:
-[`POST:/oauth2/token`](https://vippsas.github.io/vipps-login-api/#/Vipps%20Log%20In%20API/oauth2Token) this is described in detail in [login's documentation](https://github.com/vippsas/vipps-login-api/blob/master/vipps-login-api.md#access-token)
+### Vipps Login access token
+
+Accessing the Login `userinfo` endpoint required the Vipps Login access token: [`POST:/oauth2/token`](https://vippsas.github.io/vipps-login-api/#/Vipps%20Log%20In%20API/oauth2Token). The client constructs the request by adding the parameters described below to the HTTP body by using the `application/x-www-form-urlencoded` format.
+
+**Request**
+
+*Headers*
+
+| Header            | Description                           |
+| ----------------- | ------------------------------------- |
+| Content-Type      | "application/x-www-form-urlencoded"   |
+| Authorization     | "Basic {Client Credentials}"          |
+
+The Client Credentials is a base 64 encoded string consisting of the Client id
+and secret issued by Vipps joined by ":"
```

Comment:
should the value joined by ":" on the client ID and secret also be evident in the example under grant_type for "client_credentials" ? on line 560

Reply:
No the value is "client_credentials" 😕 like the actual string "client_credentials" I can update the documentation below to make this more clear.

Reply:
"Client Credentials" with capital Cs? I think not. I think we should also write `client_id` and `client_secret` explicitly, to avoid possible confusion.

Reply:
What about {base 64 encoded string based on client_id and client_secret}? with the explanation below on how you generate that